dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerExportTests.java

```diff
@@ -383,6 +384,17 @@ public void shouldReturnFilteredEvent()
             .body( "event", equalTo( "ZwwuwNp6gVd" ) );
     }
 
+    @Test
+    void shouldReturnInvalidPropertyWhenOrderOnLegacyCreatedField()
```

Comment:
Can you please add

`// TODO(tracker): remove with old tracker`

so we remember to remove it one day 😅